pkg/graph/schema/types/mto_milestone.graphql

```diff
@@ -12,17 +12,53 @@ type MTOMilestone {
   # DB Fields
   id: UUID!
   name: String! # If sourced from a Common Milestone, this will resolve from the common milestone table (you can't edit this for common milestones!)
+  key: CommonMilestoneKey
+  mtoCommonMilestoneID: UUID
+  mtoCategoryID: UUID
```

Comment:
Will this technically never be null, or do we not return anything here in the case of `uncategorized`?

If it's the former, could you update the type to be
```suggestion
  mtoCategoryID: UUID!
```

If it's the latter, I think a comment here would be helpful (since that's not how it behaves on the actual Category/SubCategory types, which always have IDs)

Reply:
Realizing that this is resolved from the models field, so I presume this _can_ be `NULL` -- can you just add a comment clarifying that this means something along the lines of:
```md
`MTOMilestone.mtoCategoryID` does _not_ always equal `MTOMilestone.Category.ID`
```

Reply:
Sure thing! Sounds good. But yeah you are correct. The Category will have UUID.Nil eg (`0000-0000-000-etc`), but the milestone itself can have a `nil` category id from the database. I'll add some clarifying comments! I think it probably is appropriate to leave this nullable as it says something about the underlying data type

Reply:
Resolved in [781550b](https://github.com/CMS-Enterprise/mint-app/pull/1454/commits/781550b7c44442a244b34acb43d26c3f9a6d31d8)